__tests__/app.test.js

```diff
@@ -94,3 +94,26 @@ describe("GET /api/articles/:article_id", () => {
       });
   });
 });
+
+describe("GET /api/articles", () => {
+  test("return an array of article objects", () => {
+    return request(app)
+      .get("/api/articles")
+      .expect(200)
+      .then((response) => {
+        const articles = response.body.article;
+        expect(articles.length).toBeGreaterThan(0);
```

Comment:
We could give a specific number here, we know how many are included in our test data.